frontend/pages/register/passport/index.tsx

```diff
@@ -0,0 +1,95 @@
+import Link from 'next/link'
+
+import { USStateSelect } from "../../../components/StateSelect/UsStateSelect"
+import { Logo } from '../../../components/Logo/Logo'
+import styles from './passport.module.css'
+
+export default function Passport({ name = "Herbert Q. Placeholder" }) {
+  function handleSubmit() {
+    console.log("submitting application!");
+  }
+  return (
+    <article className={styles.container}>
+      <Logo />
+      <h1 className={styles.heading}>Passport Account Application</h1>
+      <div className={styles.paragraph}>
+        <p>
+          Hello <em>{name}</em>, thank you for your continued interest in the National Police Data
+          Coalition.
+        </p>
+        <p>
+          We are able to provide access to legally protected data to users with the appropriate
+          permissions. This form will submit your profile for approval.
+        </p>
+      </div>
+      <form>
```

Comment:
no need for form

Reply:
So, I don't agree with this. Even if it passes a11y checks on Lighthouse, using some other container is much worse from a semantic point of view. This component takes several pieces of input from the user and will (eventually) send it to the back-end, there is no better role for it than that of form.

Reply:
That is all true however from my experience form element has introduced too many side effects and doesn't need to follow the semantic point of view for the risk of side effects involved. We have the component already maintaining the data entered in and do not need the HTML inputs and forms to rely on storing the data, so it fits another separation of concerns by keeping all the core logic in the JS and HTML only for structure.

I am willing to discuss further and we can bring this to the rest of the team as well and see their thoughts, but in my experience using the form element, even for the semantic benefit, just adds too much complexity for the simplicity of the component. Please let me know if you want to further discuss since I will need some convincing on this subject and always a great opportunity to div into something deeper for us to learn. :)

Reply:
To be clear, is your objection to having a <form> tag in the codebase period, or are you assuming that I'm writing this with the attention of submitting the input data via that form element? Because if it's the latter, that's not my intention.

Reply:
hmmm So then why use the form? I think my objective aligns with not using <form> at all

Reply:
For semantic and a11y reasons, which going against creates more problems for us than it would solve.

Reply:
oh to use <form> for accessibility reasons (a11y?) hmmm after doing some googling (https://gomakethings.com/why-use-a-form-element-when-submitting-fields-with-javascript/) looks like you are on the right track and they are more important than I thought. Just to clarify, will only be using for semantic purposes and for a11y reasons and not going to use the built in other stuff with it? In the past that is where I have run into too many issues with using a form but if just for semantic use and a11y bare minimum stuff then sounds good to me.

Reply:
Great discussion!

Reply:
Yeah, I'm not going to say I'll NEVER submit code that isn't idiomatic to the React way of doing things, but that's pretty high on my list of what I think about when writing. Submitting data via form elements is definitely on the list of "don't do this".